instrumentation/http_client/lib/opentelemetry/instrumentation/http_client/instrumentation.rb

```diff
@@ -11,6 +11,8 @@ module Instrumentation
     module HttpClient
       # The Instrumentation class contains logic to detect and install the HttpClient instrumentation
       class Instrumentation < OpenTelemetry::Instrumentation::Base
+        include InstrumentationHelpers::HTTP::RequestAttributes
```

Comment:
```suggestion
        include InstrumentationHelpers::HTTP::InstrumentationOptions
```

Reply:
i'm removing the `InstrumentationOptions` portion of this for now,  i think there's a few related issues and the scope (add better config option declaration and documentation syntax, add deprecation support, add env var for config option support, etc) has enough scope to warrant a larger PR, and i don't want that to block the request attribute helpers work